libs/model-ad/util/src/lib/navigation-links.ts

```diff
@@ -0,0 +1,42 @@
+import { NavigationLink } from '@sagebionetworks/explorers/models';
+
+export const headerLinks: NavigationLink[] = [
+  {
+    label: 'Home',
+    routerLink: [''],
+    activeOptions: { exact: true },
+  },
+  {
+    label: 'Model Overview',
+    routerLink: ['model-overview'],
+  },
+  {
+    label: 'Gene Expression',
+    routerLink: ['gene-expression'],
+  },
+  {
+    label: 'Disease Correlation',
+    routerLink: ['disease-correlation'],
+  },
+];
+
+export const footerLinks: NavigationLink[] = [
+  {
+    label: 'About',
+    routerLink: ['about'],
+  },
+  {
+    label: 'Help',
+    url: 'https://help.adknowledgeportal.org/apd/Model-AD+Explorer+Resources.4077682781.html',
```

Comment:
Can we use the `helpUrl` constant here? If not, do we still need the `helpUrl` constant or could it be removed?

```suggestion
    url: helpUrl,
```

Reply:
I've added both to constants.  Thanks!